pom.xml

```diff
@@ -44,6 +44,12 @@
             <version>31.0.1-jre</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Looks like you can remove the `test` scope in the guava definition above and then remove this dependency entry.

Reply:
code change applied